plotly/offline/__init__.py

```diff
@@ -7,5 +7,8 @@
     download_plotlyjs,
     init_notebook_mode,
     iplot,
-    plot
+    iplot_mpl,
+    plot,
+    plot_mpl,
+    plotly_takeover
```

Comment:
(tbh I agree-- in case this reference got lost  from the original pr it's based off of [this function] (https://github.com/jakevdp/mpld3/blob/202792bcddbfe957ade08a5085691ab176a4563d/mpld3/_display.py#L363) )

maybe `enable_mpl_offline` ?